.github/workflows/code-hygiene.yml

```diff
@@ -12,3 +12,12 @@ jobs:
 
       - name: Linelint
         uses: fernandrone/linelint@0.0.4
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run ShellCheck
+        uses: ludeeus/action-shellcheck@master
+        with:
+          ignore_paths: ./gradlew
```

Comment:
It seems like this exclusion is not matching:

```
 Warning: ./gradlew:168:65: warning: args5 is referenced but not assigned. [SC2154]
Warning: ./gradlew:169:74: warning: args6 is referenced but not assigned. [SC2154]
Warning: ./gradlew:170:83: warning: args7 is referenced but not assigned. [SC2154]
Warning: ./gradlew:171:92: warning: args8 is referenced but not assigned. [SC2154]
...
Excluded: ! -path *./.git/* ! -path *.go ! -path */mvnw ! -path *././gradlew/* ! -path */./gradlew/*
```

Probably we need:
```
ignore_paths: */gradlew
```

Reply:
Maybe I understood the official docs a bit wrong.

```
    - name: Run ShellCheck
      uses: ludeeus/action-shellcheck@master
      with:
        ignore_paths: ignoreme ignoremetoo
        ignore_names: ignorable.sh
````

This will skip sample/directory/with/files/ignoreme/test.sh, sample/directory/with/files/ignoremetoo/test.sh and sample/directory/with/files/ignorable.sh.

But maybe it works different with hidden files, let's see if with ``*/gradlew``-> ``"*/gradlew"`` works better!

https://github.com/marketplace/actions/shellcheck